example/exporter/kube-state-metrics/service.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    prometheus.io/scrape: 'true'
+  name: kube-state-metrics
+  labels:
```

Comment:
I made this same mistake. I missed that `kube-state-metrics` exposes metrics with a `namespace` label, while the config that the Prometheus Operator generates also automatically adds the `namespace` label, which causes colliding `namespace` labels, which Prometheus itself handles somewhat gracefully but causes confusion because one of the labels get renamed to `exported_namespace`. The solution was a lately introduced feature to disable the Prometheus Operator from relabelling that `namespace` onto the target, using the an annotation like this:

```
alpha.monitoring.coreos.com/non-namespaced: "true"
```

I used it [here](https://github.com/coreos/kube-prometheus/pull/33/files#diff-11b5648b58a69c037faec93a547d6f17R8). I'm hoping to merge https://github.com/coreos/kube-prometheus/pull/33 soon, which will be soon after the `v0.6.0` release of the Prometheus Operator, which should be some time this week.